modules/itemManager.ts

```diff
@@ -164,7 +164,9 @@ class ItemManager {
     price: number = 0,
     section: string
   ) {
-    const itemId = await db.items.add({
+    const itemId = crypto.randomUUID();
+    await db.items.add({
+      id: itemId,
```

Comment:
I ended up still using the Web Crypto API - because I found that switching IndexDB to use string ids (`&id` syntax in Dexie) that the creation of an ID was not automated as it was before with `++id` 

I'm not sure if I'm using the wrong implementation here, and I did see notes in the Dexie "syncable" docs with other syntax to assign IDs. 

But this does work, items have a much more unique ID being generated now, and considering we're making shopping lists here I'm not going to stress too much about this.